populate-db-with-masks.py

```diff
@@ -0,0 +1,25 @@
+from random import choice
+from base64 import b64encode
+from app.app import make_app
+from app.models import PPE
+
+app, db, migrate, bootstrap, login = make_app(__name__)
```

Comment:
This was the helper file I mentioned earlier. It populates the database with some mask types for testing. In the future, we could expand it to do more data population to make testing easier (ex. set up 50 hospitals with tons of valid exchanges).

Reply:
Obviously, this didn't get used much.